test/tests/functional/pbs_hook_debug_input.py

```diff
@@ -91,7 +92,8 @@ def test_queuejob_hook_debug_input_has_queue_name(self):
         attr = {ATTR_qtype: 'execution', ATTR_enable: 'True'}
         self.server.manager(MGR_CMD_CREATE, QUEUE, attr, id=new_queue)
 
-        input_file_pattern = 'hook_queuejob_%s*.in' % hook_name
+        input_file_pattern = os.path.join(self.server_hooks_tmp_dir,
+                                          'hook_queuejob_%s*.in' % hook_name)
         self.remove_files_match(input_file_pattern)
```

Comment:
This join is required because, the du.listdir lists files with absolute path (unlike the earlier os.listdir) and there is a fnmatch() inside remove_files_match() looking for a match. Match would fail if there was no join. Yes, I checked that the file gets removed.
Sure. I will add a check on the file removal inside the function.